coderd/autobuild/executor/lifecycle_executor_test.go

```diff
@@ -2,7 +2,6 @@ package executor_test
 
```

Comment:
Review: I ended up making a lot of changes in this test because I had been using a simple `* * * * *` schedule which is lower than the now-enforced default 🙃